conan/api/subapi/new.py

```diff
@@ -85,7 +86,9 @@ def _read_files(self, template_folder):
     @staticmethod
     def render(template_files, definitions):
         result = {}
+        name = definitions.get("name", "Pkg")
```

Comment:
Do not default to a package with upper case, upper case names are no longer allowed in 2.0

Reply:
It was not the name of the package but like the python class name that we camel-cased in develop, the name of the package is name, anyway I can default it to a lowercase